tests/utils/normalizeclipboarddata.js

```diff
@@ -17,4 +17,44 @@ describe( 'normalizeClipboardData', () => {
 			normalizeClipboardData( '<span class="Apple-converted-space"> </span>x<span class="Apple-converted-space">\u00a0</span>' )
 		).to.equal( ' x ' );
 	} );
+
+	it( 'should strip all spans with no attributes', () => {
```

Comment:
I was wondering whether this is cleanup should be really done.

On one hand the data is *correct* and, hypothetically, there could be a plugin that creates empty spans and want them retained when copied from editor to editor.

On the other hand, in 99.9% of cases empty span is just a garbage because spans are used mostly to hold formatting. Also a feature that creates just an empty span is probably a bad practice and something that may blow up :).

So I think this is fine.